### PR TITLE
UI-login

### PR DESCRIPTION
**Pull request by Kaito-Kido**
# Description
UI for login, signup page, and newfeed
# Screen shot
image
image
image

### CODE REVIEW

File: app/views/header/_header.slim [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.header-container 
+    .logo-search-bar
+        a href = "/"
```
Comment (nathan-phan-goldenowl) [inline app/views/header/_header.slim:3]: prefer using link to
```
link_to root_path do
```

File: app/views/posts/index.slim [hunk 2]
```diff
@@ -0,0 +1,120 @@
+= render "/header/header"
+
+.contain
+    .column-post
+        
+        .post
+            .post-top
+                .avatar
+                    img src="https://res.cloudinary.com/practicaldev/image/fetch/s--R7e2_9ta--/c_fill,f_auto,fl_progressive,h_90,q_auto,w_90/https://dev-to-uploads.s3.amazonaws.com/uploads/user/profile_image/671687/7242818e-37e5-4696-939c-8d81e821a376.jpeg" alt="avatar"
```
Comment (nathan-phan-goldenowl) [inline app/views/posts/index.slim:9]: prefer using `image_tag` for img

File: app/views/users/login.slim [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    .login-box 
+        .registration-content  
+            h1 
+                | Welcome to DEV Community 👩‍💻👨‍💻
+            .registration-description 
+                p 
+                    | DEV Community 👩‍💻👨‍💻 is a community of 968,547 amazing developers
+        .registration-actions 
+            .link-login-btn.apple
+                <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" role="img" aria-labelledby="a1xr86zvn0zz4m6m5iqp179h7gg3yxqs" aria-hidden="true" class="crayons-icon"><title id="a1xr86zvn0zz4m6m5iqp179h7gg3yxqs">apple</title>
```
Comment (nathan-phan-goldenowl) [inline app/views/users/login.slim:13]: make it an assets file and serve it as an image use `image_tag` here

File: app/views/users/new.slim [hunk 4]
```diff
@@ -0,0 +1,37 @@
+= render "/header/header"
```
Comment (nathan-phan-goldenowl) [inline app/views/users/new.slim:1]: Don't repeat. move this line into `layouts/application.slim`

File: app/views/header/_header.slim (hunk 1 above)
Comment (Kaito-Kido) [reply app/views/header/_header.slim:3]: image

**Review by nathan-phan-goldenowl — APPROVED** [review]

**Pull request merged**